source/content/guides/support/07-faq.md

```diff
@@ -20,13 +20,13 @@ reviewed: "2020-12-13"
 
 Yes. For more details, talk to your Customer Success Manager or [contact sales](https://pantheon.io/contact-us).
 
-### Is there a support number we can call? If so, is this service available for every package?
+### Is there a support number we can call? Is this service available for all account tiers?
 
-Diamond and Platinum customers have access to Pantheon On-Call, which includes an emergency 800-number to page an on-call engineer. For more details, [contact sales](https://pantheon.io/contact-us).
+Diamond and Platinum customers can call the 24/7 Premium Support Hotline for urgent account-related issues. You can find the phone number in the Support tab of your workspace.
```

Comment:
The phone hotline is not limited to urgent issues, it's for all issues and _also_ to submit an emergency ticket if needed. Suggestion:
> Diamond and Platinum customers can call Pantheon's 24/7 Premium Support Hotline directly for any technical issues, escalations, site, billing, or overage queries. You can find the phone number in the Support tab of your workspace. Note that this is strictly for filing a ticket, and you will not reach a support engineer by using this method.